app/bower_components/angular-cryptography/README.md

```
@@ -0,0 +1,74 @@
+
+##Usage
+
+* add module dependency ('mdo-angular-cryptography') to angular
+```js
+var demoApp = angular.module('app', ['services', 'mdo-angular-cryptography']);
+```
+
+* setup the encryption key in your config
+```js
+angular.module.('app').config(['$cryptoProvider', function($cryptoProvider){
+	$cryptoProvider.setCryptographyKey('ABCD123');
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)